oleesir/diaryReload
# #160870195 User should be able to see a sign in page

#### What does this PR do?
This pull request adds a sign in template to the app
#### Description of Task to be completed?
- add signin template
#### How should this be manually tested?
- Navigate to the UI folder and open signin.html in your browser
- You will see a sign in template
#### What are the relevant pivotal tracker stories?
[#160870195](https://www.pivotaltracker.com/story/show/160870195)
#### Any background context you want to add?
N/A
#### Screenshots
<img width="1280" alt="screen shot 2018-09-29 at 7 07 45 pm" src="https://user-images.githubusercontent.com/17679511/46249082-04e3d000-c41b-11e8-96c7-388c9c23f508.png">
